src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -11,7 +11,10 @@ import viper.gobra.ast.frontend.{PReceive, PSendStmt}
 import viper.gobra.reporting.Source.Verifier
 import viper.gobra.util.Constants
 import viper.gobra.util.Violation.violation
-import viper.silver.ast.SourcePosition
+import viper.silver.ast.{FalseLit, SourcePosition}
+import viper.silver.plugin.standard.refute.{Refute, RefuteError, RefuteErrorReason}
+import viper.silver.plugin.standard.smoke.SmokeDetectionInfo
+import viper.silver.verifier.{ErrorMessage, ErrorReason, errors, reasons}
```

Comment:
Do we need all of these imports? In particular, we do not seem to be using `FalseLit` or `SmokeDetectionInfo`

Reply:
-> [2bd1f22](https://github.com/viperproject/gobra/pull/776/commits/2bd1f22e9cf908143bae93c9a4730a6abcf094e6)